src/main/java/com/kodilla/ecommercee/controller/GroupController.java

```diff
@@ -0,0 +1,42 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.GroupDto;
+import com.kodilla.ecommercee.mapper.GroupMapper;
+import com.kodilla.ecommercee.service.DbService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("v1/group")
+@CrossOrigin("*")
```

Comment:
W produkcyjnych apkach, które wymagają uwierzytelniania raczej się nie pisze @CrossOrigin("*") bo to jest potencjalnie niebezpieczne. Wyobraź sobie sytuację, że bank zrobi @CrossOrigin("*"). Ja mogę przygotować stronkę (swoją domową czy firmową) tak, żeby w tle skrypty JS wysyłały żądania w tle do największych banków w polsce z żądaniem transferu pieniędzy na moje konto. Jeśli użytkownik jest zalogowany do banku w czasie gdy przegląda moją stornkę firmową/domową to takie żądanie nie będzie w żaden sposób odróżnialne od normalnej akcji użytkownika. W taki sposób moja stronka może wykonywać akcje za użytkownika na innych stronach (w jego imieniu). Są inne mechanizmy zabezpieczające przed takimi wektorami ataku jak kody jednorazowe (z doświadczenia wiem, że ludzie i tak nie patrzą na kwotę i nr rachunku docelowego) czy tokeny CSRF. Dla bezpieczeństwa jednak (jeśli nie jest to stronka public-domain bez uwierzytelniania) nie wystawiałbym @CrossOrigin("*"). Na potrzeby tego kursu możesz jednak tak zostawić.

Reply:
Dziękuję za tę uwagę - będę pamiętał o tym. Ja pamiętam, że dodałem to dlatego, że nie chciało mi się coś skompilować - już nie pamiętam co to było. Ale gdzieś znalazłem rozwiązanie, gdzie po dodaniu zaczęło działać. A tym razem dodałem to można powiedzieć mechanicznie - po prostu skopiowałem z zadania z kursu.